.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
I think the repo should already have a .gitignore file so you shouldn't need to add this.